flake.nix

```diff
@@ -35,32 +45,47 @@
           ];
         };
 
+        # Note: In-case this is changed, dream2nix's nodejs setting must also be changed
+        nodejs = pkgs.nodejs-16_x;
         pkgs_static = pkgs.pkgsCross.musl64;
 
-        bp =
-          pkgs.callPackage nix-npm-buildpackage { nodejs = pkgs.nodejs-12_x; };
-        npmPackages = bp.buildNpmPackage {
-          src = ./.;
-          npmBuild = "echo ok";
-        };
+        sidechain = (dream2nix-lib.makeFlakeOutputs {
+          source = ./.;
+
+          packageOverrides = {
+            webpack-cli = {
+              remove-webpack-check = {
+                patches = [ ./nix/patches/remove-webpack-check.patch ];
+              };
+            };
+          };
+
+          inject = { acorn-import-assertions."1.8.0" = [[ "acorn" "8.7.1" ]]; };
+
+          settings = [{ subsystemInfo.nodejs = "16"; }];
+        }).packages.${system}.sidechain;
+
+        npmPackages =
+          builtins.attrValues self.packages.${system}.sidechain.dependencies;
```

Comment:
```suggestion
        npmPackages =  builtins.attrValues ((dream2nix-lib.makeFlakeOutputs {
          source = ./.;

          packageOverrides = {
            webpack-cli = {
              remove-webpack-check = {
                patches = [ ./nix/patches/remove-webpack-check.patch ];
              };
            };
          };

          inject = { acorn-import-assertions."1.8.0" = [[ "acorn" "8.7.1" ]]; };

          settings = [{ subsystemInfo.nodejs = "16"; }];
        }).packages.${system}.sidechain);
```

Reply:
Since `sidechain` is the whole project it's not a super nice name, do we need to have it as a separate key?

Reply:
Just need to get ahold of self.packages.${system}.sidechain 
 
To use to point to the node modules path

Reply:
This should be resolved, i cleared things up in the last commit.